planner/core/exhaust_physical_plans.go

```diff
@@ -1809,6 +1809,11 @@ func (p *LogicalJoin) shouldUseMPPBCJ() bool {
 	return checkChildFitBC(p.children[0]) || checkChildFitBC(p.children[1])
 }
 
+// canPushToCop checks if it can be pushed to some stores.
+func (p *LogicalJoin) canPushToCop(storeTp kv.StoreType) bool {
+	return len(p.NAEQConditions) == 0 && p.baseLogicalPlan.canPushToCop(storeTp)
```

Comment:
There will be some compatible issues here:
Before this pr, for sql like `select * from t where nullable_col not in (select xx from t1)`, if the subquery meet the broadcast threshold, it can be pushed to TiFlash, but after this pr, there is no way to pushdown such join to TiFlash? I think before TiFlash's support of naaj is ready, it's better to add a switch to let user decide to use naaj or cross join

Reply:
great

Reply:
Since it will be controlled by the switch before tiflash supports the exec logic. Just forbid pushing down for now is okay?

Reply:
yes，after the switch，the len(p.NAEQConditions) == 0 should exactly be 0 here